renovate-presets/default.json5

```diff
@@ -60,13 +66,8 @@
       followTag: 'next',
       separateMajorMinor: false,
       schedule: ['at any time'],
-      matchDepNames: [
-        '@angular-devkit/**',
-        '@angular/**',
-        '@schematics/**',
-        'angular/**',
-        'ng-packagr',
-      ],
+      matchDepNames: ['@angular-devkit/**', '@angular/**', '@schematics/**', 'ng-packagr'],
+      matchPackageNames: ['angular/**'],
```

Comment:
What's the difference between both options? is this because there is no display/dep name for the actions?

Reply:
The `matchDepNames` uses the dependency name, in case of a WORKSPACE, this is the `name` property of the toolchain. `matchPackageName` uses the GitHub org/repo.

Reply:
Gotcha, thx for the insight